src/components/BMDashboard/MaterialsList/MaterialsTable.jsx

```diff
@@ -45,7 +45,9 @@ export default function MaterialsTable({ filteredMaterials }) {
   const sortingAsc = columnName => {
     let sorted = [];
     if (columnName === 'ProjectName') {
-      sorted = [].concat(...sortedData).sort((a, b) => (a.project.name >= b.project.name ? 1 : -1));
+      sorted = []
+        .concat(...sortedData)
+        .sort((a, b) => (a.project?.name >= b.project?.name ? 1 : -1));
```

Comment:
**nit**: Could [toSorted](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted) be clearer and more performant?  Not sure what browser versions we target.